mail-filter/popfile/popfile-1.1.3-r1.ebuild

```diff
@@ -29,21 +31,19 @@ RDEPEND="virtual/perl-Digest-MD5
 		dev-perl/IO-Socket-SSL
 		dev-perl/Net-SSLeay )
 	xmlrpc? ( dev-perl/PlRPC )"
-
-DEPEND="app-arch/unzip"
-
-S="${WORKDIR}"
+BDEPEND="app-arch/unzip"
 
 src_prepare() {
+	default
 	local f
-	for f in `find ./ -type f`; do
+	for f in $(find ./ -type f || die); do
```

Comment:
We should really grep for backticks and find other cases of these

Reply:
Yeah, I did check and there are couple more cases